operations/operations/company-cadence.md

```diff
@@ -118,8 +118,9 @@ Every 6 months we'll be asking staff to spend around 3 minutes completing a [sta
 
 Links to previous staff enablement surveys:
 
-* [March 13, 2019 staff enablement survey](https://docs.google.com/forms/d/1RwHyliyW6-Hh87K0L9srNB2qpDdgnu1_THcgg4lfgRs/edit#response=ACYDBNgZGE2_rGr-IPo126aHqHfDHQHdZWRjJ-vw-blT4iu6PV-FKUNf51bK7-vl9s3wll0)
-* [September 18, 2019 staff enablement survey](https://docs.google.com/forms/d/15ezikBYd4HGbhP9RiZrmCyBGjMEo_G-V8DQAKKqa6wM/edit)
+* [March 2019 staff enablement survey](https://docs.google.com/forms/d/1RwHyliyW6-Hh87K0L9srNB2qpDdgnu1_THcgg4lfgRs/edit#response=ACYDBNgZGE2_rGr-IPo126aHqHfDHQHdZWRjJ-vw-blT4iu6PV-FKUNf51bK7-vl9s3wll0)
```

Comment:
@natalie-hub The first links to the survey question form, the other two to the survey results. Confirming that was intended?

Reply:
@jasonblais - I hadn't originally updated that link since that's what was there previously. However, after your comment it does feel a bit weird to leave in there in a less digestible format. I updated the link to lead to the exact slide from the March 2020 deck that shared the historical info from March 2019.